fairlearn/reductions/_constant_predictor.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+class ConstantPredictor:
+    def __init__(self, constant_value):
+        self._constant_value = constant_value
+
+    def predict(self, *args, **kwargs):
```

Comment:
Actually, this is bad. The predictor needs to respond to array-valued argument correctly. I'll have to fix this.